binding/Binding/GRVkBackendContext.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace SkiaSharp
+{
+	public class GRVkBackendContext : SKObject, ISKReferenceCounted
+	{
+		[Preserve]
+		internal GRVkBackendContext (IntPtr h, bool owns)
+			: base (h, owns)
+		{
+		}
+
+		public static GRVkBackendContext Assemble (
```

Comment:
See some comments in https://github.com/mono/skia/pull/69, but I would rather have the overload that takes delegates - but if this is useful, then we can keep this one.

Reply:
I can say it's quite useful in a case like ours, where we interop between C++ and C# and share the vulkan instance between the two, while also creating a separate vulkan queue for skia, to allow the UI to render in a separate thread.